scripts/yaml_to_cmake.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''Tool for a simple parsing of YAML files and return a ;-list of <key>=<value>
+pairs to use within a CMake build file.
```

Comment:
but this is still only meant for use with west, right?

Reply:
> but this is still only meant for use with west, right?

The idea is that users who do not want to use west to manage projects can set ZEPHYR_MODULES directly at CMake time.

Reply:
This script is intended to be a general way of converting a yaml file containing key-value maps into a `;`-list that can be used by CMake.
Right now it can convert a `module.yml` file with the embedded schema, but there is no reason it cannot be extended to take a schema-file as argument, and thus convert an yaml-file.